src/page-content/navigation/header.json

```
@@ -61,22 +61,22 @@
       "mainMenu": [
         {
           "title": "Documentation",
-          "description": "Documentation for all Ory products",
+          "description": "Open Source Documentation",
           "to": "/docs"
         },
         {
           "title": "Community",
-          "description": "Be part of the Ory Community",
+          "description": "Be part of Ory Community",
```

```suggestion
          "description": "Be part of the Ory Community",
```

"Sei ein Teil der Gemeinschaft"
vs.
"Sei ein Teil Gemeinschaft"

🤔